admin/schedule-form-excludes.php

```diff
@@ -298,12 +298,15 @@
 							$size = $site_size->filesize( $file );
 
 							if ( false !== $size ) :
-								$size          = $size ?: '0';
-								$excluded_size = $excluded_site_size->filesize( $file ) ?: '0'; ?>
+								$size          = $size ?: 0;
+								$excluded_size = $excluded_site_size->filesize( $file ) ?: 0;
```

Comment:
Much better!

I think we could remove the `B` but not hugely important, we could do that separately in order to ship this more quickly if you want.